nc/source/ecs/EntityComponentSystem.cpp

```diff
@@ -100,7 +101,8 @@ bool EntityComponentSystem::DoesEntityExist(const EntityHandle handle) const noe
 
 bool EntityComponentSystem::DestroyEntity(EntityHandle handle)
 {
-    if (DoesEntityExist(handle))
+    /** Friendly reminder - this invalidates m_active iterators */
```

Comment:
Will you move this comment to line 101 (right above the method sig) so that it is clear it is not referring to !DoesEntityExist